pkg/system/config.go

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright 2023 cuisongliu@qq.com.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package system
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/containers/buildah"
+	"github.com/containers/storage/pkg/homedir"
+)
+
+type Config interface {
+	Get(string) (string, error)
+	Set(string, string) error
+}
+
+type systemConfig struct{}
+
+func (c *systemConfig) Get(key string) (string, error) {
+	return getValueOrDefault(key)
+}
+
+func (c *systemConfig) Set(key, value string) error {
+	return setValue(key, value)
+}
+
+func NewConfig() Config {
```

Comment:
initialize a global config getter/setter so caller don't need to invoke function `NewConfig` repeatedly.

Reply:
done